beacon-chain/db/validator.go

```diff
@@ -72,9 +97,29 @@ func (db *BeaconDB) DeleteValidatorIndex(pubKey []byte) error {
 	})
 }
 
+// HasValidators returns true if all validator in a list of public keys are in the bucket.
+func (db *BeaconDB) HasValidators(pubKeys [][]byte) bool {
```

Comment:
Sorry to be pedantic here, but can we rename this to `HasAllValidators` so that it is clear it only returns true if all validators are in the set?

Reply:
changed HasValidators to return true if one validator exists

Reply:
changed `HasAllValidators` to return true if one validator exists